src/main/java/carpet/utils/SpawnOverrides.java

```diff
@@ -83,16 +83,6 @@ public static boolean isStructureAtPosition(ServerLevel level, ResourceKey<Struc
         return level.structureManager().getStructureAt(pos, fortressFeature).isValid();
     }
 
-    public static boolean isStructureAtPosition(ServerLevel level, StructureType<?> structure, BlockPos pos)
-    {
```

Comment:
Yeah I left this one cause the structure API was changing a lot at that point. yeet I guess